packages/grid/x-data-grid-pro/src/DataGridPro/DataGridPro.tsx

```diff
@@ -481,11 +481,15 @@ DataGridProRaw.propTypes = {
    * @param {MuiEvent<MuiBaseEvent>} event The event that caused this prop to be called.
    */
   onCellEditStop: PropTypes.func,
+  /**
+   * Callback fired when a cell gains focus.
+   * @param {GridCellParams} params With all properties from [[GridCellParams]].
+   */
+  onCellFocusIn: PropTypes.func,
   /**
    * Callback fired when a cell loses focus.
    * @param {GridCellParams} params With all properties from [[GridCellParams]].
    * @param {MuiEvent<MuiBaseEvent>} event The event object.
-   * @param {GridCallbackDetails} details Additional details for this callback.
```

Comment:
I've noticed that this param doesn't really exist:

https://github.com/mui/mui-x/blob/cb1ca1f7b2c404954e2840e62792f786e96f8657/packages/grid/x-data-grid/src/hooks/features/focus/useGridFocus.ts#L43-L47

Reply:
It is added inside `publishEvent`